tests/application/admin/test_views.py

```diff
@@ -392,3 +392,26 @@ def test_admin_user_can_see_all_data_sources(self, test_app_client, logged_in_ad
 
         assert "Police statistics 2019" in page.text
         assert "2011 Census of England and Wales" in page.text
+
+    def test_data_sources_ordered_lexicographically_by_title(self, test_app_client, logged_in_admin_user):
+        DataSourceFactory.create(title="Police statistics 2019")
+        DataSourceFactory.create(title="2011 Census of England and Wales")
+
+        response = test_app_client.get("/admin/data-sources")
+        page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
+
+        main = page.find("main")
+        assert main.text.index("2011 Census of England and Wales") < main.text.index("Police statistics 2019")
+
+    def test_form_has_checkbox_for_each_data_source(self, test_app_client, logged_in_admin_user):
+        ds1 = DataSourceFactory.create(title="Police statistics 2019")
+        ds2 = DataSourceFactory.create(title="2011 Census of England and Wales")
+
+        response = test_app_client.get("/admin/data-sources")
+        page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
+
+        form = page.find("form", action=url_for("admin.data_sources"))  # TODO: Update to `admin.merge_data_sources`
+        assert form
+
+        checkboxes = form.find_all("input", type="checkbox")
```

Comment:
You could use the existing helper to check that the checkboxes have the right labels?

```
assert find_input_for_label_with_text(form, "Police statistics 2019").get('type') == 'checkbox'
```
?

Reply:
That matches text exactly for the label, which doesn't work for this right now because the checkbox labels contain the title, publication date and URL.

But maybe after splitting that into label+hint, it would work, yep.